scripts/apps/users/views/edit-form.html

```diff
@@ -221,6 +221,50 @@ <h5>{{ user.username}}</h5>
                 </div>
             </div>
 
+            <div sd-info-item class="form__row" ng-if="privileges.users">
+                <div class="sd-line-input">
+                    <label class="sd-line-input__label" for="facebook">{{ 'Facebook'|translate }}</label>
```

Comment:
you can use `::` for translations, eg. `{{ :: 'Facebook' | translate }}` - it will only eval this once and not on every digest